### ISSUE TITLE
Dynamically load different form schemas to collection 

### ISSUE THREAD

**Issue by almogdesign** [aldeed/meteor-autoform#576]
Been testing out autoform which is pretty amazing. I wanted to know if it's possible to load different form schemas dynamically under one collection or must I use a Non-Collection Forms. 

My current approach which is not working I'm changing the ID which is works fine
```javascript
{{> quickForm collection="Forms" id="{{formSchemaName}}" type="insert"}}   
```

How would I attach different schema based on the id?
```javascript
Forms.attachSchema(new SimpleSchema({
    author: {
        type: String,
        label: "Author"
    },
    created: {
        type: Date,
        label: "created",
        optional: true
    }
})); 
```

I also if there is a better approach for this? Or is the best approach using a a Non-Collection Form?

**Comment by almogdesign** [comment]
I'm closing this I did a different work around

**Comment by jasli2** [comment]
@almogdesign Can you share your work around? I had this requirement also.

**Comment by almogdesign** [comment]
@jasli2 I ended up not using this and just loading different templates

**Comment by aldeed** [comment]
I've done some work in the devel branch to make `schema` attribute reactive. Hopefully will be released as 5.0 within a week or so.

**Comment by jasli2** [comment]
@aldeed Great. Waiting for the release. 

Right now, I am using a schema function to specify schema dynamically in client side and expose a meteor method in server side to change schema which attached to the collection. 

Looking forward to the official solution.   :)
